chainer/links/connection/bilinear.py

```diff
@@ -60,32 +60,36 @@ def __init__(self, left_size, right_size, out_size, nobias=False,
         # This initialization is a modification of
         # that of Linear function.
 
+        shape = (left_size, right_size, out_size)
         if isinstance(initialW, (numpy.ndarray, cuda.ndarray)):
-            assert initialW.shape == self.W.shape
-        initializers.init_weight(self.W.data, initialW)
+            assert initialW.shape == shape
+        self.add_param('W', shape,
+                       initializer=initializers._get_initializer(initialW))
 
         if not self.nobias:
-            self.add_param('V1', (left_size, out_size))
-            self.add_param('V2', (right_size, out_size))
-            self.add_param('b', out_size)
-
+            V1_shape = (left_size, out_size)
+            V2_shape = (right_size, out_size)
+            b_shape = (out_size,)
             if isinstance(initial_bias, tuple):
                 V1, V2, b = initial_bias
```

Comment:
How about using `initialV1`, `initialV2`, and `initialb` instead of `V1`, `V2`, and `b`, respectively, for more readable code?